code/modules/cooking/recipes/cultural/recipes_human.dm

```diff
@@ -235,3 +274,44 @@
 	items = list(/obj/item/reagent_containers/food/snacks/whitechocolate)
 	reagent_mix = RECIPE_REAGENT_REPLACE
 	result = /obj/item/reagent_containers/food/snacks/bluemoon
+
+//Eridani
+
+/singleton/recipe/puffpuffs
+	appliance = SKILLET
+	reagents = list(/singleton/reagent/nutriment/flour = 10, /singleton/reagent/water = 5, /singleton/reagent/spacespice = 2, /singleton/reagent/sodiumchloride = 1)
+	reagent_mix = RECIPE_REAGENT_REPLACE
+	result = /obj/item/reagent_containers/food/snacks/bowl/puffpuffs
+
+/singleton/recipe/fufu
+	appliance = SAUCEPAN | POT
+	fruit = list ("banana" = 2)
```

Comment:
Follow the convention of the file you are editing. If list is not followed by a space in the rest of the file, then don't add one.
```suggestion
	fruit = list("banana" = 2)
```